### ISSUE TITLE
Latest installer release based on old Mbed CLI and GCC versions

### ISSUE THREAD

**Issue by maclobdell** [ARMmbed/mbed-cli-windows-installer#40]
On the Mbed website, all new users get directed towards this page to install Mbed CLI.
https://os.mbed.com/docs/mbed-os/v5.14/quick-start/offline-with-mbed-cli.html
For Windows users, this links to the release page on this repo.  At the time of this ticket, the latest release is based on Mbed CLI and GCC that were in use in December of 2018  (a full year ago).  

If cloning the latest Mbed OS version, this results in the following concerning warning.  

```
[Warning] @,: Compiler version mismatch: Have 6.3.1; expected version >= 9.0.0 and < 10.0.0
```

Please create a new installer release with latest updates to the version of Mbed CLI and GCC associated with the latest Mbed OS release.  Currently 5.14.  Soon to be 5.15.  

Note this is the same issue that was highlighted in https://github.com/ARMmbed/mbed-cli-windows-installer/issues/28, which was closed without explanation.  

Temporary workaround for users that find this issue - 

1. Update mbed cli with the following command
```
pip install -U mbed-cli --user
```

2. Update GCC
Download and install latest v9 at 
https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads

**Comment by tdjastrzebski** [comment]
@maclobdell @arekzaluski Do I correctly understand that is is the only and official off-line mbed-os compile tool and it is practically abandoned?

**Comment by cblack-sve** [comment]
yeah... if its not Pelion, Arm doesn't seem to care about it anymore
